### DISCUSSION TITLE
[✨] Allow preventing Form action

### DISCUSSION THREAD

**Discussion by wmertens** [QwikDev/qwik-evolution#121] [Proposals For Qwik]
### Is your feature request related to a problem?

When JS is enabled and you're using a Form only to redirect, it would save a server roundtrip to redirect immediately.

### Describe the solution you'd like

action only fires after a custom onSubmit$ and only if the event is not prevented.

This is what the docs AI expects as the API

### Describe alternatives you've considered

special attributes

### Additional context

preventdefault is already used to allow action to run locally, so it will mean passing a fake event to onSubmit$ and checking if the default was prevented.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
